components/plugin_marketplace/marketplace_item/marketplace_item.js

```diff
@@ -372,40 +424,55 @@ export default class MarketplaceItem extends React.Component {
             );
         }
 
-        const pluginDetailsInner = (
-            <>
-                {this.props.name} <span className='light subtitle'>{versionLabel}</span>
-                {localTag}
-                <p className={classNames('more-modal__description', {error_text: this.props.error})}>
-                    {this.props.error ? this.props.error : this.props.description}
-                </p>
-            </>
-        );
+        let labelsTags;
+        if (this.props.labels && this.props.labels.length !== 0) {
+            labelsTags = this.props.labels.map((label) => (
+                <Label
+                    key={label.name}
+                    name={label.name}
+                    description={label.description}
+                    url={label.url}
+                />
+            )
+            );
+        }
 
-        let pluginDetails;
+        let pluginName;
         if (this.props.homepageUrl) {
-            pluginDetails = (
+            pluginName = (
                 <a
                     aria-label={ariaLabel}
                     className='style--none more-modal__row--link'
                     target='_blank'
                     rel='noopener noreferrer'
                     href={this.props.homepageUrl}
                 >
-                    {pluginDetailsInner}
+                    {this.props.name}
                 </a>
             );
         } else {
-            pluginDetails = (
+            pluginName = (
                 <span
                     aria-label={ariaLabel}
                     className='style--none'
                 >
-                    {pluginDetailsInner}
+                    {this.props.name}
                 </span>
             );
         }
 
+        const pluginDetails = (
+            <>
+                {pluginName}
+                <span className='light subtitle'>{versionLabel}</span>
+                {localTag}
```

Comment:
I propose we remove knowledge of the `local` tab from the frontend, and have the server inject it into the labels payload.

Reply:
+1 and it can take care of the case where the `EnableRemoteMarketplace` is disabled and the `LOCAL` is out of context. See https://community-release.mattermost.com/core/pl/7sbuergejtgmi8t91r7a4xs89e

Reply:
I agree with the proposal, but would like to tackle the change in a different PR as this requires both server and webapp changes. If you are fine with this @lieut-data @marianunez, I would open a ticket and assign it to me.

Reply:
Created https://mattermost.atlassian.net/browse/MM-21611

Reply:
I will include the changes in this PR, as it's fairly easy to implement.

Reply:
See d0c8d962f4f6bf5e545b130a09775178979cc859

Reply:
Is there a server PR for handling the local tag logic?

Reply:
@marianunez https://github.com/mattermost/mattermost-server/pull/13425